app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -212,33 +209,31 @@ class TeamsAndRepositoriesConnector @Inject()(
     http.GET[Seq[Team]](teamsAndServicesBaseUrl + s"/api/teams")
 
   def allDigitalServices(implicit hc: HeaderCarrier): Future[Seq[String]] =
-    http.GET[Seq[String]](
-      url = teamsAndServicesBaseUrl + s"/api/digital-services",
-      queryParams = teamsAndRepositoriesArchived.toSeq)
+    http.GET[Seq[String]](teamsAndServicesBaseUrl + s"/api/digital-services")
 
   def teamInfo(teamName: TeamName)(implicit hc: HeaderCarrier): Future[Option[Team]] =
     http
-      .GET[Option[Team]](
-        url = teamsAndServicesBaseUrl + s"/api/teams_with_details/${encodePathParam(teamName.asString)}",
-        queryParams = teamsAndRepositoriesArchived.toSeq)
+      .GET[Option[Team]](teamsAndServicesBaseUrl + s"/api/teams_with_details/${encodePathParam(teamName.asString)}")
       .recover {
         case _ => None
       }
 
   def teamsWithRepositories(implicit hc: HeaderCarrier): Future[Seq[Team]] =
-    http.GET[Seq[Team]](
-      url = teamsAndServicesBaseUrl + s"/api/teams_with_repositories",
-      queryParams = teamsAndRepositoriesArchived.toSeq)
+    http.GET[Seq[Team]](teamsAndServicesBaseUrl + s"/api/teams_with_repositories")
 
   def digitalServiceInfo(digitalServiceName: String)(implicit hc: HeaderCarrier): Future[Option[DigitalService]] = {
     val url = teamsAndServicesBaseUrl + s"/api/digital-services/${encodePathParam(digitalServiceName)}"
     http.GET[Option[DigitalService]](url)
   }
 
   def allRepositories(implicit hc: HeaderCarrier): Future[Seq[RepositoryDisplayDetails]] =
+    http.GET[Seq[RepositoryDisplayDetails]](teamsAndServicesBaseUrl + s"/api/repositories")
+
+  def archivedRepositories(implicit hc: HeaderCarrier): Future[Seq[RepositoryDisplayDetails]] =
```

Comment:
could create a func `def repositories(archived: Option[Boolean])` then can delegate `allRepositories` and `archivedRepositories` to it (or even just call directly passing the relevant `archived` param)

Reply:
Done. I prefer to keep it as two functions externally as I think it is more self-documenting that way.